Assets/MixedRealityToolkit/_Core/Inspectors/MixedRealityControllerMappingProfileInspector.cs

```diff
@@ -26,13 +26,13 @@ public class MixedRealityControllerMappingProfileInspector : MixedRealityBaseCon
         private static GUIContent[] actionLabels;
         private static int[] actionIds;
 
+        private SerializedProperty renderMotionControllers;
+        private SerializedProperty useDefaultModels;
+        private SerializedProperty overrideLeftHandModel;
+        private SerializedProperty overrideRightHandModel;
+
         private void OnEnable()
         {
-            if (!CheckMixedRealityManager())
```

Comment:
we should probably keep this check here seeing as we're asking for the manager a few lines later.